blog-article.html

```diff
@@ -0,0 +1,358 @@
+        />
+        <h2>Enior Apple Employee Alleges Sexism At</h2>
+        <p>GAMING</p>
+
+        <p>
+          <img src="./img/btn-icons/person-icon.png" alt="" />
+          Tony Nguyen
+        </p>
+        <time class="blog-date">
+          <img src="./img/btn-icons/calendar-icon.png" alt="" />
+          20 Jun 2022
+        </time>
+
+        <p class="blog-story">
+          The Basilisks that players collect are represented with NFTs on
+          Ethereum’s blockchain with real-world value. Players use the Basilisks
+          they have gathered to battle other players to win ether (ETH). In
+          Balthazar Dragons, players are immersed in a 3D open world to explore
+          and capture dragon-like beasts called Basilisks. Balthazar Dragons is
+          an upcoming fantasy role-playing game developed on the Ethereum
+          blockchain by a decentralized autonomous organization (DAO) called the
+          Balthazar DAO.
+        </p>
+        <figure>
+          <blockquote>
+            “The first huge metaverse arena to arrange live sports, hang out and
+            even make bets and wagers”
+          </blockquote>
+          <figcaption>
+            <p class="blog-article-rating">&starf; 8.5/ 10</p>
+            <cite>Markout Corporation</cite>
+          </figcaption>
+        </figure>
+
+        <!-- group pics with tag figure -->
+        <figure>
+          <img src="./img/img-blog/img-blog1.png" alt="Green dragon head" />
+          <img
+            src="./img/img-blog/img-blog2.png"
+            alt="Warrior is standing infront of giant green dragon"
+          />
+          <figcaption style="display: none">Balthazar Dragons</figcaption>
+        </figure>
+
+        <h3>NFTs are digital assets that represent real-world objects</h3>
+        <p>
+          Setting up your avatar and personalizing your nickname on Binance is
+          quick and easy. Users can go to binance.com under Dashboard > Settings
+          > My Profile > Avatar and Nickname. For those of you who own NFT
+          assets, you can select the NFT you want to use from your collection as
+          your avatar to personalize your profile. For users who do not own any
+          NFT avatars, check out the NFT avatar collections page to get started.
+        </p>
+        <p>
+          Alternatively, if you’re on nft.binance.com, go to User Center >
+          Settings > Basic > Set Avatar and Nickname, and you’ll be redirected
+          to the main site. Binance NFT Creators have the option of setting
+          their Avatars on nft.binance.com. Once you set your new unique
+          nickname, it will be standardized and used across the Binance
+          ecosystem.
+        </p>
+        <h3>NFT Avatar Generators</h3>
+        <p>
+          You can find a range of NFT marketplaces where you can buy existing
+          NFTs or list your own NFTs for sale.
+        </p>
+
+        <!-- Tags and share block -->
+        <dl>
```

Comment:
Not so correct way to use `dl` here.
This tag is useful when you want to create a list that pairs terms with their corresponding definitions or descriptions. 

examples: 

> "Luna" is a song by Colombian singer-songwriter Feid

dl 
 dt "Luna"
 dd **is** a song by Colombian singer-songwriter Feid

> The Big Duck is a [ferrocement](https://en.wikipedia.org/wiki/Ferrocement) building in the shape of a duck.

dl 
 dt The Big Duck 
 dd  **is** a [ferrocement](https://en.wikipedia.org/wiki/Ferrocement) building in the shape of a duck.

### When to Use the `<dl>` Tag

1. **Glossaries:** When you need to present a glossary of terms and their definitions.
2. **FAQ Sections:** For structuring frequently asked questions (questions and answers).
3. **Metadata Lists:** To display lists of metadata or attributes and their corresponding values.
4. **Any Paired Information:** When you need to show any kind of paired information where each item has a corresponding description.

### When Not to Use `<dl>`

- **Non-Paired Information:** If the information isn’t paired (like a simple list of items), use `<ul>` or `<ol>`.
- **Tabular Data:** For tabular data, use the `<table>` tag instead of `<dl>`.
- **Complex Layouts:** For very complex layouts, other HTML elements and CSS may be more appropriate.

```html
<dl>
  <dt>HTML</dt>
  <dd>HyperText Markup Language, the standard markup language for creating web pages.</dd>
  
  <dt>CSS</dt>
  <dd>Cascading Style Sheets, a style sheet language used for describing the presentation of a document written in HTML or XML.</dd>
  
  <dt>JavaScript</dt>
  <dd>A programming language commonly used to create interactive effects within web browsers.</dd>
</dl>
```